packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.controller.js

```diff
@@ -138,11 +167,14 @@ export default class {
       () => {
         this.goBack();
       },
-      () => {
+      (response) => {
```

Comment:
Could be written into a `.catch` instead.

Reply:
Commit amended with correction